modules/python/init.zsh

```diff
@@ -93,7 +93,9 @@ if (( $+VIRTUALENVWRAPPER_VIRTUALENV || $+commands[virtualenv] )) && \
   VIRTUAL_ENV_DISABLE_PROMPT=1
 
   # Enable 'virtualenv' with 'pyenv'.
-  if (( $+commands[pyenv] && $+commands[pyenv-virtualenv-init] )); then
+  if (( $+commands[pyenv] )) && \
+     pyenv commands | command grep -q virtualenv-init
```

Comment:
You're right. My mistake. I wasn't paying attention to the -q... also pipelines in if statements usually trip me up. I tend to think it's the first command in the pipeline, but it's actually the last... one of the reasons I try to avoid pipes in conditionals if possible is that it's easy to mess that up.